packages/fluentui/react/src/components/Popup/Popup.tsx

```diff
@@ -506,38 +411,42 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
     );
   };
 
-  dismissOnScroll = (e: Event) => {
-    this.trySetOpen(false, e);
+  const dismissOnScroll = (e: TouchEvent | WheelEvent) => {
+    trySetOpen(false, e);
   };
 
-  trySetOpen(newValue: boolean, eventArgs: any) {
-    const isOpenedByRightClick = newValue && isRightClick(eventArgs);
+  const trySetOpen = (
+    newValue: boolean,
+    event: React.KeyboardEvent | React.MouseEvent | KeyboardEvent | MouseEvent | TouchEvent | WheelEvent
```

Comment:
Someday we should enable [`no-implicit-any`](https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options)...